### PR TITLE
Support for JUnit 5

### PR DESCRIPTION
**Pull request by softqwewasd**

### CODE REVIEW

File: test/pom.xml [hunk 1]
```diff
@@ -17,6 +17,11 @@
             <artifactId>rxjava</artifactId>
         </dependency>
 
+        <dependency>
```
Comment (flowertwig) [inline test/pom.xml:20]: We should remove this if not needed.

**Review by flowertwig — COMMENTED** [review]
I think we should migrate one or two test classes to junit5 just to show that it works.

**Review by rassmate — APPROVED** [review]
GJ!

File: test/src/test/java/se/fortnox/reactivewizard/examples/junit5_examples/Junit5Tests.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.function.Supplier;
+
+class Junit5Tests {
```
Comment (jepp3) [inline test/src/test/java/se/fortnox/reactivewizard/examples/junit5_examples/Junit5Tests.java:11]: I think we should remove these pure example test cases. We can state in the reactive wizard documentation in backstage that junit5 is supported out of the box maybe.

**Review by jepp3 — CHANGES_REQUESTED** [review]
I think we should remove these pure example test cases. We can state in the reactive wizard documentation in backstage that junit5 is supported out of the box maybe.

**Comment by softqwewasd** [comment]
The example code has been removed. I migrated a test class in the _jaxrs_ module from JUnit 4 to JUnit 5 since I could not find tests in the _test_ module that are fit to be migrated into an example parameterized test.

**Review by jepp3 — APPROVED** [review]

**Review by flowertwig — APPROVED** [review]

File: test/src/test/java/se/fortnox/reactivewizard/test/LoggingInfoVerifierTest.java [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import org.apache.logging.log4j.Level;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
```
Comment (rassmate) [inline test/src/test/java/se/fortnox/reactivewizard/test/LoggingInfoVerifierTest.java:6]: unused import?

**Review by rassmate — APPROVED** [review]

**Pull request merged**